tests/test_smu.py

```diff
@@ -216,8 +232,15 @@ def test_smu(  # noqa: PLR0915
         assert str(value) in stdout
 
     filepath = f"./temp_test_{sys.version_info.major}{sys.version_info.minor}.csv"
-    try:
+
+    # TODO: remove this deprecation check in v3
+    with mock.patch(
+        "tm_devices.drivers.pi.tsp_device.TSPDevice.export_buffers", mock.MagicMock()
+    ), pytest.warns(DeprecationWarning, match=r"Use export_buffers\(\.\.\.\) instead"):
         smu.write_buffers(filepath, "smua.nvbuffer1")
```

Comment:
You could also add an assertion in here to make sure the mock object was called once.

Reply:
willl do